FE/babel.config.json

```
@@ -0,0 +1,15 @@
+{
+  "presets": [
+      [
+          "@babel/preset-env",
+          {
+           "targets" : { 
+               "ie":11
+           },
+           "useBuiltIns" : "usage",
+           "corejs" : {"version": 3, "proposals": true }
```

사소한 부분이지만 객체 표기 targets 처럼 프로퍼티당 한 칸 씩 띄어주고 있는데 맞춰서 작성해주시면 좋을 것 같네요 
```suggestion
           "corejs" : {
               "version": 3, 
               "proposals": true
         }
```